main.css

```diff
@@ -0,0 +1,769 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 2;
+}
+
+.bar-one {
+  display: flex;
+  justify-content: space-between;
+  background-color: black;
+}
+
+.bar-one ul {
+  display: flex;
+  list-style: none;
+  margin: 0.5em 1em;
+}
+
+.bar-one li,
+.bar-two li {
+  margin: 0 0.6em;
+}
+
+.removable-icons {
+  display: none;
+}
+
+.bar-one a {
+  text-decoration: none;
+  font-size: 0.8em;
+  color: #999;
+}
+
+.bar-two {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+}
+
+.removable-img {
+  width: 120px;
+  display: none;
+}
+
+.bar-two ul {
+  display: flex;
+  list-style: none;
+  margin: 1em 1em;
+}
+
+.bar-two a {
+  text-decoration: none;
+  font-size: 0.9em;
+  color: #999;
+}
+
+.list-1 li {
+  margin-top: 0.2em;
+}
+
+.list-2 li {
+  margin: 0 0.2em;
+}
```

Comment:
As requested by the previous TSE, please use the DRY principle to optimize your CSS file